### DISCUSSION TITLE
Deploy without repo

### DISCUSSION THREAD

**Discussion by patrickhoehn** [deployphp/deployer#2375] [Help needed] [answered]
Is it possible in V7 to deploy without a repository? I'm building on my local machine and then just upload the files. When I comment out the set repository command, I get this error `fatal: repository '/www/htdocs/user/site/releases/14' does not exist` with `error  in update_code.php on line 107:` and `exit code 128 (Invalid exit argument)`

**Comment by patrickhoehn** [comment, marked as answer]
Achieved this with overwriting deploy:update_code task

**Reply by adithyasrinivasan** [reply]
Hi Patrick,

I'm facing the same issue right now, could you please elaborate?

**Reply by patrickhoehn** [reply]
Hi @adithyasrinivasan, I just added this code to my deploy.php:
```
task('deploy:update_code', function () {
    // Do nothing
});
```
and removed the set repository part
